### ISSUE TITLE
Internalionalize the tooltips of the logging options

### ISSUE THREAD

**Issue by Gallaecio** [wesnoth/wesnoth#2837]
The tooltips that you see when you hover one of the check boxes at ‘Preferences → Advanced → Logging’ as currently not internationalized.

```
adrian@curros:~/proxectos/wesnoth/1.14$ grep "Info level logging" * -FIRi
data/gui/window/logging.cfg:                                                                                            {_GUI_LOGGER_RADIOS {_INFO} "Info level logging: more information"}
adrian@curros:~/proxectos/wesnoth/1.14$
```

It should be possible to translate the for strings that act as tooltips for those check boxes.

**Comment by sevu** [comment]
> It is since recently

       --log-level=domain1,domain2,...
              sets the severity level of the log domains.  all can be used to match any log domain. Available levels: error, warning, info, debug.  By default the error
              level is used.

How is the new logging level called?

**Comment by CelticMinstrel** [comment]
@sevu If I recall correctly, I updated the manpage on master (and that update is also included in the list of commits that need backporting for 1.14.1). The new level is `none`.

**Comment by shikadiqueen** [comment]
I'm handling the addition of a UI element for the log disable option in #2972.

**Comment by sevu** [comment]
Looking at this again: 
The manpage tells `By default the error level is used.`
The UI about which this issue is has `warning` set by default.

I assume there are is no difference between the two and it's warning for both?

**Comment by shikadiqueen** [comment]
Correct. Fixed in commit 350865bffc7cc7beb920f1ec9046e7170fe5d332.
